src/instructions.cpp

```diff
@@ -27,7 +27,7 @@ void instructions::rnd_reg_byte(chip8::registers& regs, instr_t instr) noexcept
 	const auto x_offset = instructions::get_lower_nibble<size_t>(instr[0]);
 
 	auto engine = std::default_random_engine{std::random_device{}()};
-	auto random_number = std::uniform_int_distribution<uint8_t>(0, 255)(engine);
+	auto random_number = static_cast<uint8_t>(std::uniform_int_distribution<int>(0, 255)(engine));
```

Comment:
PS 
`IntType  - The result type generated by the generator. The effect is undefined if this is not one of short, int, long, long long, unsigned short, unsigned int, unsigned long, or unsigned long long.`
https://en.cppreference.com/w/cpp/numeric/random/uniform_int_distribution